AudioBar/AudioBarViewController.swift

```diff
@@ -96,6 +108,43 @@ public final class AudioBarViewController: UIViewController, StoreDelegate {
         }
     }
 
+    private func configureCommandCenterCommands() {
+        commandCenter.playCommand.addTarget { [weak store] _ in
+            guard
+                let store = store,
+                store.view.isPlayPauseButtonEnabled else { return .noSuchContent }
+            guard store.view.playPauseButtonEvent == .userDidTapPlayButton else { return .commandFailed }
+            store.dispatch(.playPauseButton(.userDidTapPlayButton))
+            guard store.view.playPauseButtonEvent == .userDidTapPauseButton else { return .commandFailed }
+            return .success
+        }
+        commandCenter.pauseCommand.addTarget { [weak store] _ in
+            guard
+                let store = store,
+                store.view.isPlayPauseButtonEnabled else { return .noSuchContent }
+            guard store.view.playPauseButtonEvent == .userDidTapPauseButton else { return .commandFailed }
+            store.dispatch(.playPauseButton(.userDidTapPauseButton))
+            guard store.view.playPauseButtonEvent == .userDidTapPlayButton else { return .commandFailed }
+            return .success
+        }
+        commandCenter.skipForwardCommand.addTarget { [weak store] _ in
+            guard
+                let store = store,
+                store.view.isSeekForwardButtonEnabled else { return .noSuchContent }
+            store.dispatch(.userDidTapSeekForwardButton)
+            return .success
+        }
+        commandCenter.skipBackwardCommand.addTarget { [weak store] _ in
+            guard
+                let store = store,
+                store.view.isSeekBackButtonEnabled else { return .noSuchContent }
+            store.dispatch(.userDidTapSeekBackButton)
+            return .success
+        }
+        commandCenter.skipForwardCommand.preferredIntervals = [.init(value: AudioBar.State.seekInterval)]
+        commandCenter.skipBackwardCommand.preferredIntervals = [.init(value: AudioBar.State.seekInterval)]
```

Comment:
Why isn't this done declaratively in `didUpdate view`? We should not poke module's internal state really. This is why I think `static` in `State`/`View`/... is a bad idea.